app/components/hero-graphic-components/GraphicScreenTopLeft.tsx

```diff
@@ -5,39 +5,42 @@ const GraphicScreenTopLeft = () => {
     <>
       <motion.path
         animate={{
-          scale: [1, 2, 1],
+          scale: [1, 2.5, 1],
           y: ["-0.15rem", "0.15rem", "-0.15rem"],
         }}
         transition={{
           duration: 4,
           ease: "easeInOut",
           repeat: Infinity,
+          delay: 1,
         }}
         d="M320.456 416.984H319.341V415.868H320.456V416.984Z"
         fill="#F7F7F7"
```

Comment:
You shouldn't use hard-coded colors. You want to place these inside your tailwind config. Be sure to update the rest of the code to follow this same pattern. You have it in lines 32, 43, etc.